docs/content/manual/release-specific/v1.1.1/disable_ipv6.md

```diff
@@ -10,18 +10,23 @@ https://github.com/longhorn/longhorn/issues/2197
 Longhorn v1.1.1 should work with IPv6 disabled.
 
 ## Scenario
-1. Install Kubernetes
-2. Disable IPv6 on all the nodes using the following commands
+1. Install Kubernetes 
+2. Disable IPv6 on all the worker nodes using the following 
 ```
-sysctl -w net.ipv6.conf.all.disable_ipv6=1
-sysctl -w net.ipv6.conf.default.disable_ipv6=1
+Go to the folder /etc/default
+In the grub file, edit the value GRUB_CMDLINE_LINUX_DEFAULT="ipv6.disable=1"
+Once the file is saved update by the command update-grub
+Reboot the node and once the node becomes active, 
+use the command cat /proc/cmdline to verify "ipv6.disable=1" is reflected in the values 
 ```
 3. Deploy Longhorn and test basic use cases.
     1. Accessing the UI for CRUD RWO/RWX volumes
     2. Use kubectl to create workloads using RWO/RWX volumes.
 4. Re-enable ipv6 on all the nodes using:
 ```
-sysctl -w net.ipv6.conf.all.disable_ipv6=0
-sysctl -w net.ipv6.conf.default.disable_ipv6=0
+By deleting the "ipv6.disable=1" from the GRUB_CMDLINE_LINUX_DEFAULT accessing grub file from
+/etc/default
+Perform a reboot and once the node comes active 
+use the command cat /proc/cmdline to verify "ipv6.disable=1" is reflected in the values
```

Comment:
Do you mean `ipv6.disable=0` ?

Reply:
Original values in the GRUB_CMDLINE_LINUX_DEFAULT:
GRUB_CMDLINE_LINUX=""
GRUB_CMDLINE_LINUX_DEFAULT="quiet splash"
As the original value was empty and after I added this :

GRUB_CMDLINE_LINUX_DEFAULT="ipv6.disable=1"
GRUB_CMDLINE_LINUX="ipv6.disable=1"

I mentioned removed the "ipv6.disable=1" from GRUB_CMDLINE_LINUX and set to the original, GRUB_CMDLINE_LINUX="".

We can also make it  GRUB_CMDLINE_LINUX="ipv6.disable=0" to disable it. Let me know if you want me to change it ipv6.disable=0